python/langsmith/_internal/_background_thread.py

```diff
@@ -36,8 +44,19 @@ class TracingQueueItem:
     """
 
     priority: str
-    action: str
-    item: Any = field(compare=False)
+    item: Union[SerializedRunOperation, SerializedFeedbackOperation]
+
+    def __lt__(self, other: TracingQueueItem) -> bool:
+        return (self.priority, self.item.__class__) < (
+            other.priority,
+            other.item.__class__,
```

Comment:
We're no longer comparing based on operation fwiw. I don't think that actually matters just noting.

Reply:
yep should be fine